test/unit/rum.test.js

```diff
@@ -125,24 +125,38 @@ describe('the RUM API', function () {
     })
   })
 
-  it('should add nonce attribute to script if passed in options', function () {
-    const nonce = '12345'
+  it('should get browser agent script with wrapping tag', function () {
```

Comment:
I'm going to add a few extra assertions to ensure the end of the timing header is correct